src/upnp_xml.cc

```diff
@@ -188,7 +188,7 @@ void UpnpXMLBuilder::renderObject(const std::shared_ptr<CdsObject>& obj, std::si
         auto meta = obj->getMetaData();
         auto [url, artAdded] = renderItemImage(virtualURL, item);
         if (artAdded) {
-            meta.emplace_back(MetadataHandler::getMetaFieldName(M_ALBUMARTURI), url);
+            obj->addMetaData(M_ALBUMARTURI, url);
```

Comment:
`addPropertyList` only reads the local copy, that's why it's added to `meta`. No goal to store albumart in metadata after searching it in the resources.